goosebit/updater/manager.py

```diff
@@ -54,11 +54,9 @@ async def get_rollout(self) -> Optional[Rollout]:
         return None
 
     async def update_config_data(self, **kwargs):
-        hw_model = kwargs.get("hw_model") or "default"
-        hw_revision = kwargs.get("hw_revision") or "default"
-        hardware = (
-            await Hardware.get_or_create(hw_model=hw_model, hw_revision=hw_revision)
-        )[0]
+        model = kwargs.get("hw_model") or "default"
+        revision = kwargs.get("hw_revision") or "default"
```

Comment:
Do we care to rename these for the `configData` endpoint?  Is `revision` or `model` used anywhere else important?  If so I'm not opposed to renaming these to keep it consistent.

Reply:
Maybe @sbabic can help us come with a good convention for attributes (“config data”). What do others use?